Makefile

```diff
@@ -158,6 +158,13 @@ cover: nocover
 	mv cover-nogen.out cover.out
 	go tool cover -html=cover.out -o cover.html
 
+.PHONY: cover-otel
+cover-otel:
+	@echo pre-compiling OTEL tests
+	@cd ${OTEL_COLLECTOR_DIR} && time go test -i $(shell cd ${OTEL_COLLECTOR_DIR} && go list ./...)
```

Comment:
I think this will give the wrong paths to `go test`, since the paths will be relative to subdir